php/classes/handlers/class-castos-handler.php

```diff
@@ -308,7 +318,7 @@ public function upload_podcast_to_podmotor( $post ) {
 
 		$this->logger->log( 'API URL', $api_url );
 
-		$featured_image_url = $this->get_featured_image( $post );
+		$featured_image_url = $this->get_podcast_image_url( $post );
 		if ( ! empty( $featured_image_url ) ) {
```

Comment:
Please leave a @todo note/comment that we should change the $post_body array key to `cover_image_url` but that can only be done after a Castos API update.

Once we perform the update to the Castos API, we can come back and update the key here.